pkg/dns/dns.go

```diff
@@ -284,9 +291,13 @@ func (kd *KubeDNS) newPortalService(service *kapi.Service) {
 		}
 	}
 	subCachePath := append(kd.domainPath, serviceSubdomain, service.Namespace)
+	host := kd.getServiceFQDN(service)
+	reverseRecord, _ := getSkyMsg(host, 0)
+
 	kd.cacheLock.Lock()
 	defer kd.cacheLock.Unlock()
 	kd.cache.setSubCache(service.Name, subCache, subCachePath...)
+	kd.reverseRecordMap[service.Spec.ClusterIP] = reverseRecord
```

Comment:
should we add reverse records for petset pods (with hostname and subdomain set AND a governing headless service?  Maybe that's yet another followup.  Can you just file an issue on that?